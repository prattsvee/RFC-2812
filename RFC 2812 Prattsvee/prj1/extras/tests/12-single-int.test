# single int

1234