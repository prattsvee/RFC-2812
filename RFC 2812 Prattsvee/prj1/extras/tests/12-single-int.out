[
  {
    "%k": "int",
    "%v": 1234
  }
]
